tests/test_output_formatter.cpp

```diff
@@ -80,11 +83,17 @@ auto construct_multiple_instances_list_reply()
     list_entry->mutable_instance_status()->set_status(mp::InstanceStatus::RUNNING);
     list_entry->set_current_release("16.04 LTS");
     list_entry->add_ipv4("10.21.124.56");
+    auto zone = list_entry->mutable_zone();
+    zone->set_name("zone");
+    zone->set_available(true);
```

Comment:
I see that set_available is true for all the cases. Can we also make one of them false to verify the "unavailable" scenario?